repository/models/framework/entity.py

```diff
@@ -112,6 +112,10 @@ def store(cls, repository, payload, content_type='application/xml'):
 
         if content_type == 'application/xml':
             helper.validate(payload, 'xml')
+        elif content_type == 'application/ld+json':
+            g = helper.validate(payload, 'json-ld')
+            payload = g.serialize()
+            content_type = 'application/xml'
 
```

Comment:
Then we don't validate it at this stage. To be honest i'm not sure why we were originally validating the xml here because it's also done in the controller.